src/main/java/uk/ac/ebi/eva/pipeline/jobs/flows/PopulationStatisticsFlow.java

```diff
@@ -6,48 +6,44 @@
 import org.springframework.batch.core.job.builder.FlowBuilder;
 import org.springframework.batch.core.job.flow.Flow;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Import;
-
-import uk.ac.ebi.eva.pipeline.jobs.CommonJobStepInitialization;
 import uk.ac.ebi.eva.pipeline.jobs.deciders.SkipStepDecider;
-import uk.ac.ebi.eva.pipeline.jobs.steps.PopulationStatisticsGeneratorStep;
-import uk.ac.ebi.eva.pipeline.jobs.steps.PopulationStatisticsLoaderStep;
+import uk.ac.ebi.eva.pipeline.jobs.steps.CalculateStatisticsStep;
+import uk.ac.ebi.eva.pipeline.jobs.steps.LoadStatisticsStep;
+import uk.ac.ebi.eva.pipeline.parameters.JobOptions;
 import uk.ac.ebi.eva.pipeline.parameters.JobParametersNames;
 
+import static uk.ac.ebi.eva.pipeline.configuration.BeanNames.CALCULATE_STATISTICS_OPTIONAL_FLOW;
+import static uk.ac.ebi.eva.pipeline.configuration.BeanNames.CALCULATE_STATISTICS_STEP;
+import static uk.ac.ebi.eva.pipeline.configuration.BeanNames.LOAD_STATISTICS_STEP;
+
 @Configuration
 @EnableBatchProcessing
-@Import({PopulationStatisticsGeneratorStep.class, PopulationStatisticsLoaderStep.class})
-public class PopulationStatisticsFlow extends CommonJobStepInitialization {
-
-    public static final String CALCULATE_STATISTICS = "Calculate statistics";
-    public static final String LOAD_STATISTICS = "Load statistics";
-    private static final String STATS_FLOW = "statsFlow";
+@Import({CalculateStatisticsStep.class, LoadStatisticsStep.class})
+public class PopulationStatisticsFlow {
```

Comment:
There isn't a non optional PopulationStatisticsFlow for now... it will be done at some point.